src/main/java/seedu/storemando/ui/ItemCard.java

```diff
@@ -24,8 +25,13 @@
      * @see <a href="https://github.com/se-edu/storeMando-level4/issues/336">The issue on StoreMando level 4</a>
      */
 
+    private static final String FXML = "ItemListCard.fxml";
+
     public final Item item;
 
+    private ItemExpiringPredicate expiredItemPredicate = new ItemExpiringPredicate((long) 0);
```

Comment:
maybe consider declare 2 final long at the start, so next time if we want to change the day can be easier, currently is like magic number :)

Reply:
This itself is a variable already. Does it still count as a magic number though? Because from my understanding, magic numbers are when it appears in the method and reader has no understanding of what the magic number represents.

Reply:
I think better to put like private static final long EXPIRED_NUM = 0 etc :)

Reply:
All right!